### DISCUSSION TITLE
Channels in a mess

### DISCUSSION THREAD

**Discussion by Gitalifr** [iptv-org/community#132] [Q&A]
Hi everybody,
I guess that the list of channel is in a total mess. 
I use the link with channel classified by country but same channels are repeadly added for many countries whereas they are not channels of this country. Plus there are some channels that are added in the wrong country. 
Is it possible to fix this issue ? 
Thank you for your help.

**Comment by MapGuy11** [comment]
> Hi everybody,
> I guess that the list of channel is in a total mess. 
> I use the link with channel classified by country but same channels are repeadly added for many countries whereas they are not channels of this country. Plus there are some channels that are added in the wrong country. 
> Is it possible to fix this issue ? 
> Thank you for your help. 

Could you tell us what country you are using?

**Reply by Gitalifr** [reply]
Most of the time we watch Moroccan channels and I can see mbc , Al Jazeera in the list whereas they are not Moroccan channels like many other channels. Same For example, I can see Al Jazeera in Angola, in Afghanistan and in many other countries. 
I use Kodi and simple PVR client to read the feed. 
There is also the "not 24/7" not and "geolocalised" mentions that appeared in front of some channels

**Comment by Gitalifr** [comment]
![16450172629242337959402167894021](https://user-images.githubusercontent.com/91804844/154272083-57a50a0d-aaca-463c-a754-f5ed7346000a.jpg)
 Example for Algérie on Kodi. 
BBC world isn't Algerian nor FRANCE 24 not rt america. 
Do you see what I mean ?

**Reply by Gitalifr** [reply]
Same screen for Andorra and almost all channels

**Reply by BellezaEmporium** [reply]
EN : This most likely contains channels that are streamed onto the Maghreb diaspora. It is not a "mess" per-se, it's a concatenation of some internationally-streamed channels, compiled with the Algerian ones.

FR : Il y a très certainement, en addition aux chaînes algériennes, des chaînes internationales diffusées dans la diaspora du Maghreb. Ce n'est pas un "foutoir" à proprement parler, il s'agit juste d'une concaténation d'autres chaînes internationales avec les chaînes algériennes.

**Reply by Gitalifr** [reply]
If you have a look at Saudi Arabia, Qatar, compris etc there is mbc, BBC etc same channels as well. It is not correct

**Reply by Gitalifr** [reply]
![16459789596204131644108470028035](https://user-images.githubusercontent.com/91804844/155890526-795c965d-5029-4e7f-b4e9-e6e6baf06216.jpg)

This is for kenya for instance

**Comment by Gitalifr** [comment]
![16459796135441349774202815773433](https://user-images.githubusercontent.com/91804844/155890974-02885e17-999e-47cf-8cac-eebd140d798f.jpg)

If you see whatever the country there are the same channels.

**Reply by BellezaEmporium** [reply]
As said, some channels that are international will be everywhere, which explains why they're on all countries.

**Comment by techstag** [comment]
[https://hindi.techstag.in/2022/03/indian-news-iptv-m3u8-github-links-2022.html](https://hindi.techstag.in/2022/03/indian-news-iptv-m3u8-github-links-2022.html) 
IpTV Indian Links
